type-definitions/dom-event.d.ts

```diff
@@ -0,0 +1,46 @@
+import {Stream} from "most";
+
+export function domEvent(event: string, node: EventTarget, capture: boolean): Stream<Event>
```

Comment:
It looks like TS supports default param values: https://www.stevefenton.co.uk/2013/05/typescript-default-parameters/

I think that matches both the es6 source and the intent more closely than an optional param. Let's go with a default value.